src/main/resources/db/migration/VS20230515_52__IJPS-52_statusHistory.sql

```diff
@@ -0,0 +1,10 @@
+
+CREATE TABLE public.status_history (
+	status_history_ID bigint not null,
```

Comment:
can we have all ID to id for consistency as the first sql script for IJPS-39 has them lowercase